Automattic/woocommerce-payments
# Remove the custom Translations Loader initialization

Fixes #3493

#### Changes proposed in this Pull Request

This PR removes the custom translations loader used to merge all language pack JSON files into one and enqueue it with each plugin JS file.

#### Additional context
Before #3486 the language packs did not contain a JSON with the translations for each of the plugin enqueued JS files (webpack generated files in the dist folder), so we relied `WC_Payments_Translations_Loader` to merge all JSON files into one, and the inject that single JSON file each time a JS file was enqueued (using the filter `load_script_translation_file` to always resolve to that single file.

#### Testing instructions
1. Go to **Settings** and change the Site Language to any [supported languages](https://translate.wordpress.com/projects/woocommerce/woocommerce-payments)
2. Go to **Dashboard > Updates** and click **Update translations** (this will be shown in the selected language :P)
3. Go to **Payments > Overview** and verify that the strings are fully translated:

<img width="713" alt="Screen Shot 2022-01-03 at 18 22 02" src="https://user-images.githubusercontent.com/407542/147981869-9ac082b9-6038-4e53-bbaf-8c3518eee01c.png">

<img width="726" alt="Screen Shot 2022-01-03 at 18 20 21" src="https://user-images.githubusercontent.com/407542/147981863-10305739-00ae-41ba-9bf7-8eda2a3d8759.png">

The strings from that page are JS only; they are not present in the `.PO` file, only in the `JSON` files. For example the string `Pending balance` is only present in the JSON file: `woocommerce-payments-es_ES-090b05cdc4c33b5b4d778c13de44d8ea.json`
